tests/valgrind_test.sh

```diff
@@ -7,29 +7,14 @@
 #      Jan Černý <jcerny@redhat.com>
 
 # set up variables
-valgrind_output=$(mktemp)
 logfile=$builddir/tests/valgrind_test.log
 error_code=66
```

Comment:
The PR moves those definitions to the wrapper, so I would remove those lines as well.